Common/include/CConfig.hpp

```diff
@@ -5729,6 +5743,42 @@ class CConfig {
    */
   const su2double* GetBody_Force_Vector(void) const { return body_force; }
 
+  /*!
+   * \brief Get information about the streamwise periodicity (None, Pressure_Drop, Massflow).
+   * \return Driving force identification.
+   */
+  unsigned short GetKind_Streamwise_Periodic(void) const { return Kind_Streamwise_Periodic; }
+
+  /*!
+   * \brief Get information about the streamwise periodicity Energy equation handling.
+   * \return Real periodic treatment of energy equation.
+   */
+  bool GetStreamwise_Periodic_Temperature(void) const { return Streamwise_Periodic_Temperature; }
+
+  /*!
+   * \brief Get the value of the artificial periodic outlet heat.
+   * \return Heat value.
+   */
+  su2double GetStreamwise_Periodic_OutletHeat(void) const { return Streamwise_Periodic_OutletHeat; }
+
+  /*!
+   * \brief Get the value of the pressure delta from which body force vector is computed.
+   * \return Delta Pressure for body force computation.
+   */
+  su2double GetStreamwise_Periodic_PressureDrop(void) const { return Streamwise_Periodic_PressureDrop; }
+  
+  /*!
+   * \brief Set the value of the pressure delta from which body force vector is computed.
+   * \param[in] delta_p - pressure difference between in- and outlet.
+   */
+  void SetStreamwise_Periodic_PressureDrop(su2double delta_p) { Streamwise_Periodic_PressureDrop = delta_p; }
```

Comment:
Please please please don't use the config for this, keep it in the solver, pass it to numerics "explicitly" (with a set something) instead of mixed with the rest of the world in CConfig.

Reply:
removed the Set function and made it a solver var 👍 And I actually agree ... the config should not be the place to make solver/or other things variables available to lots of places just because it is convenient